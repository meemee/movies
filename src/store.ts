import { configureStore } from '@reduxjs/toolkit';
import DevTools from './components/DevTools';
import rootReducer from './reducers';

const enhancers = [];
if (process.env.NODE_ENV === 'development') enhancers.push(DevTools.instrument())

const store = configureStore({
  reducer: rootReducer,
  devTools: {},
  enhancers: enhancers
})

export default store