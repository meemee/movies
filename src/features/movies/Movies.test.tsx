import React from 'react';
import { render } from '@testing-library/react';
import Movies from './';
import { Provider } from 'react-redux';
import store from '../../store';


describe('SearchBox', () => {
    it('renders', async () => {
        const { container } = render(
            <Provider store={store}><Movies /></Provider>
        );
    
        expect(container).toMatchSnapshot();
      });

});