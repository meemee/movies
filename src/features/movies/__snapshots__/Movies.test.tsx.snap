// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox renders 1`] = `
<div>
  <div
    class="ant-row"
    style="margin: -16px -8px 16px -8px;"
  />
</div>
`;
